torchx/lib/torchx/backend.ex

```diff
@@ -0,0 +1,219 @@
+defmodule Torchx.Backend do
+  @behaviour Nx.Backend
+
+  defstruct [:ref]
+
+  alias Torchx.NIF
+
+  def torch_type({:u, 8}), do: :byte
+
+  def torch_type({:u, 16}),
+    do: raise(ArgumentError, "PyTorch does not support unsigned 16 bit integer.")
```

Comment:
Nitpicks:

1. Use Torchx instead of PyTorch
2. Error messages in Elixir do not finish with "."

Suggestion:

```suggestion
    do: raise(ArgumentError, "Torchx does not support unsigned 16 bit integer")
```

Please change the other ones accordingly! :heart: